src/components/content-tab-import-export.tsx

```diff
@@ -180,7 +182,7 @@ const ImportSite = ( props: { selectedSite: SiteDetails } ) => {
 				<InitialImportButton
 					isInitial={ isInitial }
 					openFileSelector={ openFileSelector }
-					disabled={ isSiteExporting }
+					disabled={ isSiteExporting || isAnySitePulling }
```

Comment:
Should we also block the buttons to export the site on `Import/Export` tab while the pulling operation is running? It seems that I am still able to export when the pull is in progress?

Additionally, I noticed that it seems that when we are pulling, we are disabling the import on all local sites. Should we keep it enabled all other sites and disable it just on the site that is in pull operation progress?

Reply:
Blocking export when pulling a site: https://github.com/Automattic/studio/pull/612/commits/e8a0a07a418e9a903e2ee7a7083d680d2757ee20

I don't think we want to support importing different sites when pulling. This could be a follow-up if we consider it necessary.

Reply:
sounds good, that seems reasonable for now 👍